inputfiles/removedTypes.jsonc

```
@@ -762,6 +769,13 @@
                     }
                 }
             },
+            "RTCPeerConnectionIceEventInit": {
+                "members": {
+                    "member": {
+                        "url": null // Safari +12 only as of 2025-04
```

```suggestion
                        "url": null // WebKit only as of 2025-04
```

(Engine names are more relevant than browser names)